account-decoder/src/parse_stake.rs

```diff
@@ -19,7 +19,7 @@ pub fn parse_stake(data: &[u8]) -> Result<StakeAccountType, ParseAccountError> {
             meta: meta.into(),
             stake: None,
         }),
-        StakeState::Stake(meta, stake) => StakeAccountType::Delegated(UiStakeAccount {
+        StakeState::Stake(meta, stake, _) => StakeAccountType::Delegated(UiStakeAccount {
             meta: meta.into(),
             stake: Some(stake.into()),
         }),
```

Comment:
im assuming the intent of flags (if more flags are ever added) is only for transient internal state tracking, so doesnt need to show up in `UiStakeAccount`

Reply:
i don't think we need to impose any use-restrictions on the flags. future additions that make sense to display will likely need handling that we can't predict today. for instance i'll claim one when i get around proposing vote account bonds, but ui display would probably be something like
```rust
flags.contains(FLAG_VOTER_BOND).then_some(delegation.vote_account)
```

Reply:
https://github.com/solana-labs/solana/pull/32033/files

^^ This is a follow up PR, which adds the stake flags to `UiStakeAccount`.